app/lib/presentation/pages/welcome_page.dart

```diff
@@ -43,53 +42,38 @@ class _WelcomePageState extends State<WelcomePage> {
 
   @override
   Widget build(BuildContext context) {
-    return PageEnclosureMolecule(
-      title: '',
-      expendChild: false,
-      child: Column(
-        children: [
-          TextAtom(
-            GlobalVariables.appName,
-            style: Theme.of(context).textTheme.displayMedium,
-          ),
-          const SeparatorAtom(variant: SeparatorVariant.relatedElements),
-          if (showSubTitle)
-            AnimatedTextAtom(
-              text: 'Learn A New Language',
-              variant: AnimatedTextVariant.typewriter,
-              onDone: animatedTextFinish,
-            )
-          else
-            const TextAtom('', variant: TextVariant.title),
-          const SeparatorAtom(),
-          const SizedBox(width: double.infinity),
-          Expanded(
-            child: Container(
-              constraints: const BoxConstraints(
-                maxWidth: 280,
-              ),
+    final size = MediaQuery.of(context).size * 0.8;
+
+    return SafeArea(
+      child: SizedBox(
+        height: size.height,
+        width: double.infinity,
+        child: Column(
+          children: [
+            TextAtom(
+              GlobalVariables.appName,
+              style: Theme.of(context).textTheme.displayMedium,
+            ),
+            const SeparatorAtom(variant: SeparatorVariant.relatedElements),
+            if (showSubTitle)
+              AnimatedTextAtom(
+                text: 'Learn A New Language',
+                variant: AnimatedTextVariant.typewriter,
+                onDone: animatedTextFinish,
+              )
+            else
+              const TextAtom('', variant: TextVariant.title),
+            const Spacer(),
+            Container(
+              constraints: const BoxConstraints(maxWidth: 280),
```

Comment:
Please do not insert any value in one place, it should come from the theme or an atom.

Create a MarginedExpandedAtom



class MarginedExpandedAtom extends StatelessWidget {
  const MarginedExpandedAtom({
    required this.child,
    super.key,
  });

  final Widget child;

```dart
  @override
  Widget build(BuildContext context) {
    return Center(
      child: Container(
        constraints: const BoxConstraints(maxWidth: 800),
        margin: AppThemeData.generalHorizontalEdgeInsets,
        child: child,
      ),
    );
  }
}

```